sound/pci/hda/patch_hdmi.c

```diff
@@ -2833,10 +2833,17 @@ static int alloc_intel_hdmi(struct hda_codec *codec)
 /* parse and post-process for Intel codecs */
 static int parse_intel_hdmi(struct hda_codec *codec)
 {
+	int retry_count = 0;
 	int err;
 
+again:
```

Comment:
please, make this a loop

Reply:
@plbossart  @lyakh Ack, I'll rework a bit. I did consider retry but based on current used in sound/, "again" beats "retry" by a small margin and in HDA code "again" is more frequenty goto target.

In general, I'm not very happy about this solution in general and expect to get feedback on this  on the list as well. I've tried dozens of different things, but only thing that works reliably and not risking the common case, is this ugly retry. But yeah, no reason to make it uglier than it is.